tests/filecheck/dialects/eqsat/eqsat_ops.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+// CHECK:       builtin.module {
+
+// CHECK-NEXT:    %v0, %v1 = "test.op"() : () -> (index, index)
+%v0, %v1 = "test.op"() : () -> (index, index)
+
+// CHECK-NEXT:    %r0 = eqsat.eclass %v0 : index
+// CHECK-NEXT:    %r1 = eqsat.eclass %v0, %v1 attributes {"hello" = "world"} : index
+%r0 = eqsat.eclass %v0 : index
+%r1 = eqsat.eclass %v0, %v1 attributes {"hello"="world"} : index
```

Comment:
@PapyChacal is there a nice custom syntax way to express the fact that the result and all the operands have the same type? Is this a custom constraint thing?

Reply:
I figured it out! Pushing